code/index.html

```diff
@@ -4,18 +4,32 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Project GitHub Tracker</title>
+  <title>Technigo Boot Camp GitHub Tracker</title>
   <link rel="stylesheet" href="./style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Merriweather&family=Open+Sans&display=swap" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
 </head>
 <body>
-  <h1>GitHub Tracker</h1>
-  <h2>Projects:</h2>
-  <main id="projects"></main>
+  <header>
+    <h1>Technigo Boot Camp GitHub Tracker</h1>
+  </header>
+  <main class="page-container">
+    
+    <section id="profile-container" class="profile-container"></section>
```

Comment:
Maybe the closing tag could be placed on the row below, like you have done with the other section-elements.